assets/babel.config.js

```diff
@@ -4,5 +4,5 @@ module.exports = {
     ['@babel/preset-react', { runtime: 'automatic' }],
     '@babel/preset-typescript',
   ],
-  plugins: ['@babel/plugin-proposal-class-properties'],
```

Comment:
'@babel/plugin-proposal-class-properties' should be included in the '@babel/preset-env' preset and was causing storybook builds to fail. I'm pretty sure removing it is safe and haven't seen any negative effects.